flexget/plugins/urlrewrite_elitetorrent.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import re
+import unicodedata
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugins.plugin_urlrewriting import UrlRewritingError
+from flexget.utils import requests
+from flexget.utils.soup import get_soup
+
+from flexget.entry import Entry
+from flexget.utils.search import normalize_unicode
+
+log = logging.getLogger('elitetorrent')
+
+
+class UrlRewriteElitetorrent(object):
+    """
+    elitetorrent urlrewriter and search Plugin.
+    """
+    
+    schema = {
+        'type': 'object',
+        'properties': {
+            'category': {'type': 'string'},
+        },
+        'additionalProperties': False
+    }
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        return (url.startswith('http://www.elitetorrent.net/torrent/'))
+
+    # urlrewriter API
+    def url_rewrite(self, task, entry):
+        entry['url'] = self.parse_download_page(entry['url'])
+
+    @plugin.internet(log)
+    def parse_download_page(self, url):
+        try:
+            page = requests.get(url).content
+            soup = get_soup(page, 'html.parser')
+            download_link = soup.findAll(href=re.compile('/get-torrent/\d+'))
+            download_href = 'http://www.elitetorrent.net' + download_link[0]['href']
+            return download_href
+        except IndexError:
```

Comment:
There are multiple excpetions that need to be checked..

Firstly requests exceptions. ie http errors and timeouts.. more info here http://docs.python-requests.org/en/latest/user/quickstart/#errors-and-exceptions

Second soup parse errors http://www.crummy.com/software/BeautifulSoup/bs4/doc/#errors-when-parsing-a-document